Brokkr/src/main/cpp/commands/SetArmPostitionForDistance.cpp

```diff
@@ -6,6 +6,15 @@
 #include <cmath>
 
 namespace{
+
+  double GetCameraDistance(double NodeHeight, double AngleTargetOffset)
+  {   
+    const double kCameraAngle = 45*(3.14152653589/180);
```

Comment:
Can we work in radians?

Reply:
Answering my own questions. The camera returns yaw, pitch, and roll in degrees.

We can choose to convert those measurements to radians as they are read or convert from degrees to radians whenever we want to make calculations.

My preference would be to do all of the math in radians and convert inputs and outputs as needed.